### DISCUSSION TITLE
Where would I look to find some examples on how to animate the ReturnResultPopup.

### DISCUSSION THREAD

**Discussion by davefxy** [CommunityToolkit/Maui#1984] [Q&A]
I want to replace my use of Mopup control withe the CommunityToolKit ReturnResultPopup. As a result I need to figure out how to animate it. I want to animate the appearing and disappearing.
Thanks

**Comment by bijington** [comment]
We don't offer the ability to change the animation of Popups like Mopups do. If that functionality is key then I'd probably suggest sticking with Mopups

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
